### PR TITLE
fix : show github README file as code for a given chapter topic #496

### PR DESCRIPTION
**Pull request by jad837**

### CODE REVIEW

File: src/pages/VideoCourse.vue [hunk 1]
```diff
@@ -311,7 +284,18 @@ export default {
...
+				// changed github url to gitcdn.link
+				let rawCodeEditorURL = this.topic.codeLink;
+				rawCodeEditorURL = rawCodeEditorURL.replace(
+					'github.com',
+					'gitcdn.link/repo'
+				);
+				let lastSlash = rawCodeEditorURL.lastIndexOf('/');
+
+				this.codeEditorURL = rawCodeEditorURL + '/master/README.md';
+				console.log(this.codeEditorURL);
```
Comment (souvikbasu) [inline src/pages/VideoCourse.vue:296]: please remove console.log statements

File: src/pages/VideoCourse.vue [hunk 2]
```diff
@@ -1,102 +1,75 @@
...
-            :on-delete="deleteComment"
-            :on-edit="editComment"
-            :on-save="saveComment"
-          />
-        </b-col>
-        <b-col md="2" />
-      </b-row>
-    </b-container>
-  </div>
+	<div class="host">
```
Comment (souvikbasu) [inline src/pages/VideoCourse.vue:2]: can you please pull latest and check that there are no formatting related changes? ideally it should show only the changes done by you

**Review by souvikbasu — CHANGES_REQUESTED** [review]
The file diff shows many formatting changes. Can you please pull latest and check that there are no formatting related changes?
Also please use the --rebase switch in git pull command so that there are no merge commits. Please remove the merge commit from the PR

**Comment by jad837** [comment]
Hi there check the PR I have done the changes.

**Pull request merged**